phpstan.neon

```diff
@@ -1,5 +1,5 @@
 parameters:
-    level: 7
+    level: 4
```

Comment:
Same as for `composer.json`. Is this required. It seems like a side effect of the phpstam version bump.
If not required, please revert